README.md

```diff
@@ -10,6 +10,11 @@ This is the TShock plugin repository. It collects links to all plugins in the ec
 ----
 
 ## List of all plugins! 🧪☕️⚡️
+* [AutoTeam](https://github.com/TerraTrapezium/AutoTeam) by [GoodPro712](https://github.com/GoodPro712)
+  * Automatically assigns players a configurable team on world join.
+  * Tested on TShock 4.5.5
+  * [Download Version 1.0.0](https://files.catbox.moe/14tcj8.dll) (`AutoTeam.dll`)
```

Comment:
scr of 14tcj8.dll ok 

```
~/Downloads via 🐍 v2.7.16 via 💎 v3.0.1
❯ shasum -a 256 /Users/particles/Downloads/14tcj8.dll
805643c409bdec9c0eebfc8a67770d2673e9b9925769a0f159b966819a0065f3  /Users/particles/Downloads/14tcj8.dll
```

ha: https://hybrid-analysis.com/sample/805643c409bdec9c0eebfc8a67770d2673e9b9925769a0f159b966819a0065f3

(honestly, did not wait for ha to complete)

![Screen Shot 2021-08-11 at 23 06 04](https://user-images.githubusercontent.com/532647/129146278-0776a5b1-46ae-407b-b1b3-2707277c5e3d.png)